app/controllers/logix_controller.rb

```diff
@@ -18,6 +18,10 @@ def gettingStarted
   def examples
   end
 
+  def featured
+    @projects = FeaturedCircuit.all.select(&:valid?).map(&:project)
```

Comment:
Why validating here ? This is expensive

Reply:
I did this initially before I had implemented the hook to delete featured_circuit entry in project.rb. I'll remove it now.